addons/point_of_sale/static/src/js/Screens/ClientListScreen/ClientLine.js

```diff
@@ -6,11 +6,16 @@ odoo.define('point_of_sale.ClientLine', function(require) {
 
     class ClientLine extends PosComponent {
         get highlight() {
-            if (this.props.partner !== this.props.selectedClient) {
-                return '';
-            } else {
-                return this.props.detailIsShown ? 'highlight' : 'lowlight';
-            }
+            return this.props.partner !== this.props.selectedClient ? '' : 'highlight';
+        }
+        showClientDetails() {
+            let obj = {
+                detail: {
+                    clickEditButton: true,
+                    client: this.props.partner
+                }
+            };
+            this.__owl__.parent.clickClient(obj);
```

Comment:
can't we remove this call to "this.__owl__.parent" and trigger an event that'll call clickCLient ?